daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/EngineTest.scala

```diff
@@ -1412,10 +1414,10 @@ class EngineTest extends WordSpec with Matchers with EitherValues with BazelRunf
         .consume(lookupContractMap.get, lookupPackage, lookupKey)
 
       tx.transaction.nodes.values.headOption match {
-        case Some(NodeFetch(_, _, _, _, _, _, key)) =>
+        case Some(Node.NodeFetch(_, _, _, _, _, _, key)) =>
           key match {
             // just test that the maintainers match here, getting the key out is a bit hairier
-            case Some(KeyWithMaintainers(keyValue @ _, maintainers)) =>
+            case Some(Node.KeyWithMaintainers(keyValue @ _, maintainers)) =>
```

Comment:
why `keyValue @ _` instead of just `keyValue`?

Reply:
`keyValue @ _` is a notation to avoid singleton variable warning. 
Sometime having name for variable in pattern matching, make the later more readable.

Reply:
It does not really improve readability in this case, so I apply the change you requested.